build.gradle

```diff
@@ -0,0 +1,35 @@
+apply plugin: 'java'
+
+jar {
+  // by default fat jar
+  baseName = 'vertx-blueprint-todo-backend-fat'
+  from { configurations.compile.collect { it.isDirectory() ? it : zipTree(it) } }
+  manifest {
+    attributes 'Main-Class': 'io.vertx.blueprint.todolist.Application'
```

Comment:
Use the vert.x launcher since it will handle command line arguments such as clustering, ha, etc...